packages/server/utils/OpenAIServerManager.ts

```diff
@@ -243,6 +244,44 @@ class OpenAIServerManager {
       return null
     }
   }
+
+  async modifyCheckInQuestion(question: string, modifyType: ModifyType) {
+    if (!this.openAIApi) return null
+
+    const prompt: Record<ModifyType, string> = {
+      EXCITING: `Transform the following team retrospective ice breaker question into something imaginative and unexpected, using simple and clear language suitable for an international audience. Keep it engaging and thrilling, while ensuring it's easy to understand.
```

Comment:
-1 either this needs a length limit, or we need to adjust the UI to go wider. if you expand the image below you'll see it starts to shove the listen icon under the meeting bar

<img width="619" alt="Screenshot 2023-12-08 at 2 11 12 PM" src="https://github.com/ParabolInc/parabol/assets/5514175/56802ab6-42e6-4a8b-848e-7860287186fd">

Reply:
really good point. added char limit to the prompt and made the layout a little bit compact. modified questions are even better now, thank you!